concepts/05 Widgets/Scheduler/010 Overview.md

```diff
@@ -165,6 +236,83 @@ If your appointments have a different structure, specify:
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="dataSource"
+            text-expr="subject"
+            start-date-expr="from"
+            end-date-expr="to" />
```

Comment:
I've noticed that there are no such fields as "subject", "from", and "to" in the data source.